### PR TITLE
Sundry fixes

### PR DESCRIPTION
**Pull request by mruwnik**

### CODE REVIEW

**Comment by zarSou9** [comment]
logic looks good to me

File: app/components/Article/article.css [hunk 1]
```diff
@@ -202,4 +205,19 @@ article .banner h3 .title {
...
     margin: 0;
   }
+
+  article .link-popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: #ff000033;
```
Comment (melissasamworth) [inline app/components/Article/article.css:215]: variable!

File: app/components/Article/article.css [hunk 2]
```diff
@@ -202,4 +205,19 @@ article .banner h3 .title {
...
+    overflow: scroll;
+  }
+  article .link-popup .footnote {
+    margin: auto;
+    background-color: white;
```
Comment (melissasamworth) [inline app/components/Article/article.css:220]: variable!

**Review by melissasamworth — APPROVED** [review]

File: app/components/Article/article.css (hunk 2 above)
Comment (mruwnik) [reply app/components/Article/article.css:220]: I chose these specially for you. i.e. so that you'd get annoyed with it and change them :D

**Pull request merged**